resources/views/components/menu/notifications.blade.php

```diff
@@ -1,19 +1,9 @@
 @auth
-    <ul class="navbar-nav">
-        <li class="nav-item">
-            <a class="nav-link" href="{{ route('activity') }}" title="{{ __('Activity') }}" data-toggle="tooltip">
-                <x-fas-bolt />
-            </a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="{{ route('notification.index') }}" title="{{ __res('notification') }}" data-toggle="tooltip">
-                <livewire:notification-icon />
-            </a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="{{ route('message.index') }}" title="{{ __res('message') }}" data-toggle="tooltip">
-                <livewire:message-icon />
-            </a>
-        </li>
-    </ul>
+    {{--<li class="nav-item">
+        <a class="nav-link" href="{{ route('activity') }}" title="{{ __('Activity') }}" data-toggle="tooltip">
+            <x-fas-bolt />
+        </a>
+    </li>--}}
+    <livewire:notification-icon class="{{$class ?? ''}}" />
+    <livewire:message-icon class="{{$class ?? ''}}" />
```

Comment:
I'm not sure if there's an easy solution for this, but I find myself trying to move diagonally from the icon to the menu item, which mouses over the Manage button and changes the menu before I can click on the notification submenu item.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/10893a1e-b2ef-400d-9dcf-a9adbcf2f80e)

Reply:
There is much less right-padding when there are no notifications
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/db41c073-0d9a-41b4-b1e9-9af884d14713)

Reply:
I found myself running into this frequently, too.
By removing the fixed width of 250px of dropdown items it would adjust its width automatically and there's maybe less of an urge to go sideways?
<img width="428" alt="Screenshot 2023-09-05 at 20 21 56" src="https://github.com/RetroAchievements/RAWeb/assets/1280590/2e09d0e7-23cb-46c7-994b-96cd1fefcfb0">
It would allow the games dropdown to scale more naturally, too, actually. It's very wide already.
<img width="914" alt="Screenshot 2023-09-05 at 20 23 50" src="https://github.com/RetroAchievements/RAWeb/assets/1280590/d32ab496-6d9c-45a2-8c49-f286ac9559fc">

Reply:
Maybe right aligned just for the notifications dropdown? It doesn't work too well for the management and profile dropdowns, but feels better for notifications:
<img width="523" alt="Screenshot 2023-09-05 at 20 49 57" src="https://github.com/RetroAchievements/RAWeb/assets/1280590/d340cc99-bbef-42ce-bcd9-1e701de2dab5">

Reply:
I think the right alignment is enough for me to move the cursor straight down instead of down and left, as that's where the words are instead of a gap my brain says isn't part of the link.